eternalcore-core/src/main/java/com/eternalcode/core/EternalCore.java

```diff
@@ -240,6 +242,10 @@ public EternalCore(Plugin plugin) {
         this.audiencesProvider = BukkitAudiences.create(plugin);
         this.miniMessage = MiniMessage.builder()
                 .postProcessor(new LegacyColorProcessor())
+                .tags(TagResolver.builder()
+                        .resolver(CenterTagResolver.RESOLVER)
+                        .resolver(TagResolver.standard())
```

Comment:
To dodaje standardowe flagi czy setuje? (pytam defakto o metodę `.flags()`) bo jak dodaje to chyba nie trzeba dodawać standard